components/ResultsPageComponents/TripCountAndTravelMethods/TripCountAndTravelMethods.jsx

```diff
@@ -66,11 +67,20 @@ export default function TripCountAndTravelMethods({ dataAboutTrips }) {
                   <PieChartComponent
                     title="Active-public-shared travel"
                     data={activePublicSharedTravel}
+                    colorList={[
+                      "#044B7F",
+                      "#9BB7CC",
+                      "#366F99",
+                      "",
```

Comment:
actually, what is the empty string for?

Reply:
haha, good find @OlaShabalina ! 🤣 

It's a lazy hack I did to make the color segments look little more 'balanced', and less dark. If I omitted the empty string, the donut chart looks like this:
![image](https://user-images.githubusercontent.com/4408259/185037574-e96b4cf9-ed89-4ee2-8e27-107143e4257d.png)

Do you prefer the chart above? I can update the chart if the one above looks better 👍🏽

Reply:
Haha, nice catch, agree with you, it's better with the skipped colour! 👌🏻